src/k8s-engine/wsl.ts

```diff
@@ -569,6 +570,26 @@ export default class WSLBackend extends events.EventEmitter implements K8s.Kuber
     });
   }
 
+  /**
+   * Starts host-resolver's vsock-host process in wsl host
+   */
+  protected async startHostResolverHost() {
+    const exe = path.join(paths.resources, 'win32', 'host-resolver.exe');
+    const stream = await Logging['host-resolver'].fdStream;
+
+    return childProcess.spawn(exe, ['vsock-host'], {
+      stdio:       ['ignore', stream, stream],
+      windowsHide: true,
+    });
+  }
+
+  /**
+   * Return the Linux path to the host-resolver executable.
+   */
+  protected getHostResolverPeerPath(distro?: string): Promise<string> {
```

Comment:
We never need `distro` here (we will only run `host-resolver` in the `rancher-desktop` distribution, not `OpenSUSE` / `Ubuntu` / etc.)